azure_functions_worker/loader.py

```diff
@@ -218,6 +218,7 @@ def load_function(name: str, directory: str, script_file: str,
                f'{os.path.exists(CUSTOMER_PACKAGES_PATH)}')
 def index_function_app(function_path: str):
     module_name = pathlib.Path(function_path).stem
+    logger.info(f'Loading module {module_name}')
```

Comment:
Is this log needed?

Reply:
Nope. Good catch. Added it for debugging. Will remove.